### DISCUSSION TITLE
Different names of the same apps.

### DISCUSSION THREAD

**Discussion by ArtemChandragupta** [SylEleuth/gruvbox-plus-icon-pack#114] [General]
Hi! I'm very happy with the state of the icon theme, it has a lot of icons and is fairly consistent. Great and beautiful work!

I use NixOS and noticed that some programs have icons for themselves in this theme but don't use them. For example, mendeley because it's called `mendeley-reference-manager`. 
![image](https://github.com/user-attachments/assets/30381f3f-5631-41ae-bf60-87df84820f1a)

I made a .desktop placeholder there I changed the `Icon=` to `mendeley-desktop`, but I think it will be good if You add this variant of a name to theme.

Or do you think that nixpkgs should change the .desktop it provides?

---

Also, there is openscad-unstable which uses specical blue icon called openscad-nightly. You can use the same svg for openscad-nightly as for openscad as placeholder, or just paint the sphere in blue. Have not found it's svg, but this is png from U@untu Sn@ps I found in internet.
![image](https://github.com/user-attachments/assets/b6152b63-978d-4f88-9ba2-f6676ceafc59)

---

And the last, I there is a [Jupyter](https://jupyter.org/) without an icon. You can use [Ipython icon](https://github.com/SylEleuth/gruvbox-plus-icon-pack/blob/503b78a6ac77f2541b7242ba10c052fd3c0e2083/Gruvbox-Plus-Dark/apps/scalable/ipython.svg#L4) for it as a placeholder, because it is Ipython in most situations.

[This is original svg logo I guess](https://github.com/jupyter/notebook/blob/main/notebook.svg).

**Comment by SylEleuth** [comment]
I've added symlink to `mendeley-reference-manager`. Is it really the same as `mendeley`? Because on Arch `mendeley-desktop` `and reference` have different packages. And I just installed both and though they require registration it looks like they are two separate programs.
Couldn't find svg icon for `openscad` so I left it how it is, for nightly I've changed it's colours. I can't install unstable version so I trust You that that's the correct name for it.
Added `Jupyter` icons.

There is a lot of missing icons for applications so if You'd have anything that is missing bring it on.

PS. Missing icons is an issue, next time please post there.
